plotting/drawhits.py

```diff
@@ -239,8 +252,8 @@ def parse_args():
 phi_binning = [int(phi_range * 2 / bw_phi), -phi_range, phi_range ] # rad binning
 if debug>1: print(f"z_binning: {z_binning}, r_binning: {r_binning}, phi_binning: {phi_binning}")
 
-layer_binning = [n_layers + 1, -0.5, n_layers + 0.5]
-if is_endcap(detector_type):
+layer_binning = [n_layers, -0.5, n_layers - 0.5]
```

Comment:
We should keep the old binning here. The idea behind it is that sometimes layer numbering starts from 1 instead of  0 (e.g. for the drift chamber).

If that wider binning causes issues downstream, we should try to accommodate for that.